lib/src/player_view.dart

```diff
@@ -17,6 +17,7 @@ class PlayerView extends StatefulWidget with PlayerViewEventHandler {
   PlayerView({
     required this.player,
     super.key,
+    this.playerViewConfig = const PlayerViewConfig(),
```

Comment:
Nit: can we move this one line up so that we have `player` and `playerViewConfig` after each other?